torch/_inductor/kernel/conv.py

```diff
@@ -257,7 +275,8 @@ def convert_1x1_conv_to_mm(x, weight, bias):
     weight = L[aten.permute](weight, [1, 0])
 
     if x.get_size()[0] != 1:
-        x = ir.ExternKernel.require_stride_order(x, channels_last_order(rank))
+        if inductor_config.conv_force_channels_last:
+            x = ir.ExternKernel.require_stride_order(x, channels_last_order(rank))
```

Comment:
If the config to require channels last is working correctly, I dont think this is necessary.

Reply:
@pragupta was able to take the comment and submit a PR which is much cleaner using layout_opt could you take a look at https://github.com/pytorch/pytorch/pull/107812

This PR will now focus on adding the device_type into compile_meta and getting our ROCm compatible triton configs